app/views/health_cards/show.pdf.erb

```diff
@@ -54,6 +54,35 @@
           </div>
         <% end %>
       </div>
+      <div class="level">
+        <div class="level-left">
+          <% if @patient.lab_results.present? %>
+            <div class="level-item">
+              <p class="title">Lab Results</p>
+            </div>
+          <% end %>
+        </div>
+      </div>
+      <div class="tile is-ancestor">
+        <% @patient.lab_results.each do |i| %>
+          <div class="tile is-parent">
+            <div class="tile is-child box">
+              <div class="block">
+                <span class="mr-2 has-text-weight-bold">Result:</span><span class="has-text-weight-bold"><%= i.result %></span>
+              </div>
+              <div class="block">
+                <span class="mr-2 has-text-weight-bold">Code:</span><span class="has-text-weight-bold"><%= i.code %></span>
+              </div>
```

Comment:
Can `i.result` and `i.code` be switched to `i.name` and `i.result_name` show that the actual name of the lab test and result are displayed in the PDF like they are in `app/views/patients/show.html.erb`?

Reply:
Also is there a reason that result and code are switched from the order they are in the non-PDF patient show page?

Reply:
resolved - accidentally switched the tags around